UI/GameSettingsScreen.cpp

```diff
@@ -287,6 +287,11 @@ void GameSettingsScreen::CreateViews() {
 	CheckBox *depthRange = graphicsSettings->Add(new CheckBox(&g_Config.bDepthRangeHack, gs->T("Depth Range Hack (Phantasy Star Portable 2)")));
 	depthRange->SetDisabledPtr(&g_Config.bSoftwareRendering);
 
+	static const char *bloomHackOptions[] = { "Off", "Safe", "Balanced", "Aggressive" };
+	PopupMultiChoice *bloomHack = graphicsSettings->Add(new PopupMultiChoice(&g_Config.iBloomHack, gs->T("Bloom Effect Smoothing"), bloomHackOptions, 0, ARRAY_SIZE(bloomHackOptions), gs, screenManager()));
```

Comment:
It's a photographic term and perhaps a little obscure if you're not either a photographer or a game graphics programmer :) So yeah, you might be right, although a lot of PC gamers probably know what it is.